package-lock.json

```diff
@@ -1,8 +1,12479 @@
 {
```

Comment:
Hi, @DavidTereshchenko, please remove this file from your pr

Reply:
ready)

Reply:
You shoud remove this changes, not existing file)

Reply:
ooops, i fixed it